core/core-backend/src/main/java/io/dataease/chart/dao/auto/entity/CoreChartView.java

```diff
@@ -621,6 +632,9 @@ public String toString() {
         ", flowMapStartName = " + flowMapStartName +
         ", flowMapEndName = " + flowMapEndName +
         ", extColor = " + extColor +
+        ", customAttrMobile = " + customAttrMobile +
+        ", customStyleMobile = " + customStyleMobile +
+        ", sortPriority = " + sortPriority +
         "}";
     }
 }
```

Comment:
代码存在以下问题和优化建议：
1. `customAttr` 和 `extColor` 变量在两个地方定义，但其中一个是多余的。
2. `sortPriority` 字段没有用于实际业务逻辑的地方，可以删除或简化。

以下是修复后的代码：

```java
/**
 * <p>图形视图实体类</p>
 *
 * @author fit2cloud
 * @since 2024-12-12
 */
@TableName("core_chart_view")
public class CoreChartView implements Serializable {

    private static final long serialVersionUID = 1L;

    /**
     * ID
     */
    private Integer id;

    // 原始字段

    /**
     * 图形属性
     */
    private String customAttr;

    /**
     * 组件样式
     */
    private String customStyle;

    /**
     * 结果过滤
     */
    private List<String> customFilter;

    // 新字段

    /**
     * 数据流起点节点名称（仅支持唯一一个）
     */
    private String flowMapStartName;

    /**
     * 数据流终点节点名称（仅支持唯一一个）
     */
    private String flowMapEndName;

    /**
     * 外部颜色
     */
    private String extColor;

    /**
     * 图形属性_移动端
     */
    private String customAttrMobile;

    /**
     * 组件样式_移动端
     */
    private String customStyleMobile;

    /**
     * 字段排序优先级
     */
    private String sortPriority;

    public Long getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getCustomAttr() {
        return customAttr;
    }

    public void setCustomAttr(String customAttr) {
        this.customAttr = customAttr;
    }

    public String getCustomStyle() {
        return customStyle;
    }

    public void setCustomStyle(String customStyle) {
        this.customStyle = customStyle;
    }

    public List<String> getCustomFilter() {
        return customFilter;
    }

    public void setCustomFilter(List<String> customFilter) {
        this.customFilter = customFilter;
    }

    public String getFlowMapStartName() {
        return flowMapStartName;
    }

    public void setFlowMapStartName(String flowMapStartName) {
        this.flowMapStartName = flowMapStartName;
    }

    public String getFlowMapEndName() {
        return flowMapEndName;
    }

    public void setFlowMapEndName(String flowMapEndName) {
        this.flowMapEndName = flowMapEndName;
    }

    public String getExtColor() {
        return extColor;
    }

    public void setExtColor(String extColor) {
        this.extColor = extColor;
    }

    public String getCustomAttrMobile() {
        return customAttrMobile;
    }

    public void setCustomAttrMobile(String customAttrMobile) {
        this.customAttrMobile = customAttrMobile;
    }

    public String getCustomStyleMobile() {
        return customStyleMobile;
    }

    public void setCustomStyleMobile(String customStyleMobile) {
        this.customStyleMobile = customStyleMobile;
    }

    public String getSortPriority() {
        return sortPriority;
    }

    public void setSortPriority(String sortPriority) {
        this.sortPriority = sortPriority;
    }

    @Override
    public String toString() {
        return "CoreChartView{" +
                "id=" + id +
                ", customAttr='" + (customAttr == null ? "null" : customAttr) + '\'' +
                ", customStyle='" + (customStyle == null ? "null" : customStyle) + '\'' +
                ", customFilter=" + (customFilter != null ? customFilter.toString() : "[]") +
                ", flowMapStartName='" + (flowMapStartName == null ? "null" : flowMapStartName) + '\'' +
                ", flowMapEndName='" + (flowMapEndName == null ? "null" : flowMapEndName) + '\'' +
                ", extColor='" + (extColor == null ? "null" : extColor) + '\'' +
                ", customAttrMobile='" + (customAttrMobile == null ? "null" : customAttrMobile) + '\'' +
                ", customStyleMobile='" + (customStyleMobile == null ? "null" : customStyleMobile) + '\'' +
                ", sortPriority='" + (sortPriority == null ? "null" : sortPriority) + '\'' +
                "}";
    }
}
```

修复后的代码去除了重复字段，并添加了对 `toString()` 方法的空值处理。